erc/ERCS/erc-7562.md

```diff
@@ -317,19 +317,24 @@ These are storage rules that help to protect the bundler from denial of service:
   This means that `Paymaster` and `Factory` contracts cannot practically be an "account" contract as well.
 * **[STO-041]** `UserOperation` may not use associated storage (of either its account or from staked entity) in a contract that is a "sender" of another UserOperation in the mempool.
 
-### Staked Entities Reputation Rules
 
-* **[SREP-010]** The "canonical mempool" defines a staked entity if it has `MIN_STAKE_VALUE` and unstake delay of `MIN_UNSTAKE_DELAY`
-* **[SREP-020]** A `BANNED` address is not allowed into the mempool.\
+### General Reputation rules
+The following reputation rules apply for both staked and unstaked entities.
+
+* **[GREP-010]** A `BANNED` address is not allowed into the mempool.\
   Also, all existing `UserOperations` referencing this address are removed from the mempool.
-* **[SREP-030]** A `THROTTLED` address is limited to:
+* **[GREP-020]** A `THROTTLED` address is limited to:
     * `THROTTLED_ENTITY_MEMPOOL_COUNT` entries in the mempool.
     * `THROTTLED_ENTITY_BUNDLE_COUNT` `UserOperations` in a bundle.
     * Can remain in the mempool only for `THROTTLED_ENTITY_LIVE_BLOCKS`.
-* **[SREP-040]** An `OK` staked entity is unlimited by the reputation rule.
+* **[GREP-040]** If an entity fails the second validation or fails bundle creation, its `opsSeen` set to `10000`, and `opsIncluded` to zero, causing it to be `BANNED`.
```

Comment:
```suggestion
* **[GREP-040]** If an entity fails bundle creation after passing 2nd validation, its `opsSeen` set to `10000`, and `opsIncluded` to zero, causing it to be `BANNED`.
```

Reply:
This rule was also wrong before this PR as an SREP.  We only insta-BAN an entity that causes bundle invalidation after passing 2nd simulation, because this causes more work and there's no legitimate reason for this to happen. Just failing 2nd validation is legitimate in small numbers (any case where a state change invalidated a UserOp).  If we do this for every invalidation, the whole reputation mechanism is unused.

Reply:
Yes, fixed